mars/utils.py

```diff
@@ -73,6 +73,11 @@
 from .lib.version import parse as parse_version
 from .typing import ChunkType, TileableType, EntityType, OperandType
 
+try:
+    import ray
```

Comment:
Should use lazy import.

Reply:
The `lazy_import` is defined after this import, using `lazy_import` will make it not located at the top import statements.